c2cgeoform/scaffolds/c2cgeoform/+package+/models/c2cgeoform_demo.py_tmpl

```diff
@@ -328,9 +328,8 @@ class Excavation(Base):
                                           7, 8, 9, 10, 11, 12, 13,
                                           14, 15, 16, 17, 18, 19, 20, 21, 22, 23,
                                           24, 25, 26, 27, 28, 29]
-                            }),
-                        })
-                    })''',
+                            },
+                    }''',
```

Comment:
We may create a python dict here and serialize it as json in the widget template